.github/workflows/pre-commit.yaml

```diff
@@ -24,19 +24,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
-
-    - name: setup micromamba
-      uses: mamba-org/setup-micromamba@v1
+    - uses: actions/setup-python@v3
       with:
-        environment-file: environment.yml
```

Comment:
I assume that the full UG env is not needed for the pre-commit action and that's why you changed to a simple python env?

Reply:
Yes. The pre-commits builds an environment with its own tools (with the versions specified in the pre-commit configuration).

Reply:
All makes sense to me, thanks for explaining.